specifications/transactions/README.md

```diff
@@ -33,12 +31,12 @@ Diem defines refunds as part of the transaction framework. The refund includes a
 ### Transfer with Subaddresses
 
 Bob would like to send Alice 10 XUS:
-* Alice loads her VASP's mobile APP and generate a PaymentURI
-* The PaymentURI contains a subaddress that uniquely identifies Alice along with additional parameters specifying sending 10 XUS
+* Alice loads her VASP's mobile APP and generate an intent identifier
+* The intent identifier contains a subaddress that uniquely identifies Alice along with additional parameters specifying sending 10 XUS
 * Alice shares this with Bob via a QR code
-* The PaymentURI opens Bob's VASP's mobile app and confirms Bob's intent to send 10 XUS
+* The intent identifier opens Bob's VASP's mobile app and confirms Bob's intent to send 10 XUS
```

Comment:
Does novi's flow look like this? intent identifier links to the novi app and confirms?